### ISSUE TITLE
Accept default fill and stroke colors for DRD elements

### ISSUE THREAD

**Issue by pedesen** [bpmn-io/dmn-js#679]
Hey modeling team!

__Is your feature request related to a problem? Please describe.__

I'm currently implementing DMN in a project including a DRD Panel. Since we have a dark mode in the project we'd like to customize the fill and stroke colors of DRD elements.

__Describe the solution you'd like__

It would be nice to be able to provide a defaultStrokeColor and defaultFillColor similar to the [implementation in bpmn-js](https://github.com/bpmn-io/bpmn-js/commit/dc1f18a4972cee2d9e1bf46a6412dbb93b28506a). Is something like this planned in the near future?

__Describe alternatives you've considered__

I tried to style the svg elements from outside. It works, but it does not feel as nice as passing default colors. Happy for any more input!

```css
.dmn-drd-container .djs-visual rect {
  stroke: #ffffff !important;
  fill: #313238 !important;
}

.dmn-drd-container .djs-label {
  fill: #ffffff !important;
}

.dmn-drd-container .djs-connection polyline {
  stroke: #ffffff !important;
}
```

__Additional context__

These are the design requirements for DRD dark mode:
![image](https://user-images.githubusercontent.com/2543743/153021918-524b9e44-3000-4663-afa5-e058edd65420.png)
(Source: https://app.zeplin.io/project/5ae9bee76332bd9e298b5d41/screen/61fd48a87a576f9b9ff25dab)

**Comment by barmac** [comment]
Hey @pedesen,

thanks for opening the issue. I believe the feature request makes sense.

> Is something like this planned in the near future?

We haven't planned it but I am happy to support you if you want to contribute.

**Comment by barmac** [comment]
This is now released via `dmn-js@12.1.0`.
